examples/apmsetrate.py

```diff
@@ -3,6 +3,8 @@
 '''
 set stream rate on an APM
 '''
+from __future__ import print_function
+from builtins import range
 
 import sys, struct, time, os
```

Comment:
struct, time, os are imported but not used.

Running flake8 or autopep8 on these files might help find other issues.